### DISCUSSION TITLE
BNG Blaster

### DISCUSSION THREAD

**Discussion by Cholinmaung1** [rtbrick/bngblaster#25] [Q&A] [answered]
I want to test BNGblaster on ubuntu. I am a student, so I want to know to do bngblaster simulation. So Please guide procress

**Comment by GIC-de** [comment, marked as answer]
First you need a working BNG which you would like to test using BNG Blaster.

**Reply by GIC-de** [reply]
Now there is Quickstart guide which works even without a BNG or any kind of hardware.
https://rtbrick.github.io/bngblaster/quickstart.html
